causalpy/tests/test_input_validation.py

```diff
@@ -223,3 +224,83 @@ def test_iv_treatment_var_is_present():
                 sample_kwargs=sample_kwargs
             ),
         )
+
+
+# Regression kink design
+
+
+def setup_regression_kink_data(kink):
+    """Set up data for regression kink design tests"""
+    # define parameters for data generation
+    seed = 42
+    rng = np.random.default_rng(seed)
+    N = 50
+    kink = 0.5
```

Comment:
Is this over-riding the king value passed to the function?

Reply:
well spotted. Fixed